setup.py

```diff
@@ -1,11 +1,11 @@
 from setuptools import setup, find_namespace_packages
 
-requirements = []
+requirements = ["allensdk", "tqdm", "numpy", "tifffile"]
```

Comment:
Ugh, sorry for the allensdk requirement, I will get it out in the next round of improvements